src/WorkflowGui/Controller/WorkflowController.php

```diff
@@ -339,9 +339,14 @@ private function getVisualization($workflow, $format): string
             throw new \InvalidArgumentException($this->trans('workflow_cmd_not_found', ['dot']));
         }
 
+        $workflowRepository = $this->repository->find($workflow);
+
+        if ($workflowRepository && !$workflowRepository['enabled']) {
```

Comment:
Possibly rewrite as `!($workflowRepository['enabled'] ?? false)`

Reply:
I did split it up in two messengers, so it become more clear for enduser